source/dagger/gameplay/ping_pong/pingpong_ball.cpp

```diff
@@ -74,7 +74,7 @@ void PingPongBallSystem::Run()
         }
         else
         {
-            t.position += (ball.speed * Engine::DeltaTime());
+            t.position += (ball.speed * Engine::DeltaTime()*ConfigureSystem::BallSpeedMultiplier()*ConfigureSystem::GameSpeedMultiplier());
```

Comment:
it's kinda debug system, but you are using it without #if defined(DAGGER_DEBUG)

I suggest you to move these BallSpeedMultiplier and GameSpeedMultiplier to some none debug(non imgui) class, and make tools(imgui) change them